docs/source/zero-code-change-logging.rst

```diff
@@ -99,3 +102,203 @@ This logging information can help you:
 * Understand why certain operations fall back to CPU
 * Debug performance issues by seeing where GPU acceleration fails
 * Optimize your code by understanding synchronization patterns
+
+Profiling
```

Comment:
Should we add some kind of note or warning to this section that profiling will affect the performance (negatively)? In other words, it's important to not compare the overall runtime performance of native code with cuml.accel _profiled_ code?